jbpm-form-modeler-core/jbpm-form-modeler-service/jbpm-form-modeler-ui/src/main/java/org/jbpm/formModeler/components/renderer/FormRenderContextManagerImpl.java

```diff
@@ -109,7 +109,7 @@ public FormRenderContext newContext(Form form, String deploymentId, Map<String,
 
     private FormRenderContext buildContext(String uid, Form form, String deploymentId, Map<String,Object> inputData, Map<String,Object> outputData, Map<String,Object> forms) {
         FormRenderContext ctx = new FormRenderContext(uid, form, inputData, outputData);
-        ctx.setDeploymentId( deploymentId );
+        ctx.setServerTemplateId(deploymentId );
```

Comment:
This is bit confusing - setting deploymentId as server template...

Reply:
@mswidersky well seen... Forgot to rename the parameter. DeploymentId is no longer used, so I changed it.